doc/sphinx/user/install/local-installation/using-candi.md

```diff
@@ -34,6 +32,15 @@ class="smallcaps">p4est</span> and Trilinos.
     configuration files in that directory, named `candi.cfg` and `local.cfg`).
     When both `candi.cfg` and `local.cfg` are present in the directory, running
     candi (see below) will by default use the configuration options in `local.cfg`.
+    
+    If you want to work without the `local.cfg`, be aware that you should enable
+    - p4est
+    - trilinos
+    - hdf5 (optional)
+    - netcdf (optional)
+    - sundials (optional)
```

Comment:
I couldn't quite figure out what the "(optional)" part here means. You're suggesting in the text above that one should enable these. If that's your suggestion, then just remove the qualification, or at least explains what happens if one takes the optional at face value and doesn't do it.

Reply:
Fair enough. Updated.